### PR TITLE
Create webviews.md

### PR DESCRIPTION
**Pull request by JonasKruckenberg**
This PR adds a page detailing information about webview versions on the 3 currently supported OS'es. The main focus is on the macOS version to webkit version table.

### CODE REVIEW

File: docs/guides/architecture/webviews.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+| Mojave      | 10.14                            | [606.2.11]       | 12.0              |                                                                |
+|             | 10.14.1                          | [606.2.104.1.1]  | 12.0.1            |                                                                |
+|             | 10.14.2                          | [606.3.4]        | 12.0.2            |                                                                |
+|             | 10.14.3                          | [606.4.5]        | 12.0.3            |                                                                |
+|             | 10.14.4                          | [607.1.40.1.5]   | 12.1              |                                                                |
+|             | 10.14.6                          | [608.1.49]       | 13.0              |                                                                |
+| Catalina    | 10.14                            | [608.2.30.1.1]   | 13.0.2            |                                                                |
```
Comment (FabianLars) [inline docs/guides/architecture/webviews.md]:
```suggestion
| Catalina    | 10.15                            | [608.2.30.1.1]   | 13.0.2            |                                                                |
```

File: docs/guides/architecture/webviews.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+## Linux (WebKitGTK)
+
+| Distro                                                         | webkitgtk | WebKit    | Safari Equivalent |
+| -------------------------------------------------------------- | --------- | --------- | ----------------- |
+| debian 9                                                       | 2.18      | [605.1.3] | 11.1              |
+| Ubuntu 18.04 (arm64, armhf, ppc64el, s390x)                    | 2.20      | [606.1.4] | 12.0              |
+| Ubuntu 20.04 (arm64, armhf, ppc64el, s390x)                    | 2.28      | [610.1.1] | 14.0              |
+| Ubuntu 18.04 (amd64, i386), centos 8 (non-stream)              | 2.32      | [612.1.6] | 15.0              |
+| debian 10                                                      | 2.34      | [613.1.1] | 15.4              |
+| Ubuntu 20.04 (amd64, i386), Ubuntu 22.04, debian 11, debian 12 | 2.36      | [614.1.6] | TP 140 (16.0)     |
```
Comment (FabianLars) [inline docs/guides/architecture/webviews.md]: Okay now that i see it, i think something like having `debian 9` and `debian 9*` with * being "With all updates applied" would be kinda cool i guess?

Also funny story, what i mentioned packages.ubuntu.com would show (but didn't) in _distro_ vs _distro_-updates (the first one being the packages when the distro version was released and the second one is the updates to it), is indeed what's happening for non x86 archs.
In other words: `(amd64, i386)` == all updates applied - `(arm64, armhf, etc)` == original version.

Comment (JonasKruckenberg) [reply docs/guides/architecture/webviews.md]: Is this comment still relevant? I added a note about the situation on Ubuntu 20.04 and older and the arches, but I have no idea how to get more info about Debian 9 🤔 feel free to add that info if you have it though 😄

File: docs/guides/webview-versions.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
+# Webview Versions
+
```
Comment (lorenzolewis) [inline docs/guides/webview-versions.md]: Maybe have a quick intro paragraph here? Explaining how Tauri interacts with WRY to use the OS's native webview (may be able to just steal some text from somewhere in Architecture)

File: docs/guides/webview-versions.md [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+## WebView2 (Edge/Chromium)
+
+Tauri uses WebView2 which is based on Microsoft Edge and therefore Chromium. WebView2 can update itself, you are guaranteed a relatively recent chromium build on all Windows targets.
```
Comment (lorenzolewis) [inline docs/guides/webview-versions.md]:
```suggestion
Tauri uses WebView2 which is what powers Microsoft Edge (and is built on Chromium). Since WebView2 can update itself through OS updates, you are guaranteed a relatively recent Chromium build on all Windows targets.
```

File: docs/guides/webview-versions.md [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+
+WebView2 is supported on Windows 7 and newer and comes preinstalled on Windows 11. On versions older than Windows 11 the installer generated by Tauri takes care of ensuring WebView2 is installed on the system.
```
Comment (lorenzolewis) [inline docs/guides/webview-versions.md]: Windows 7 32 and 64 bit right? We don't need to distinguish here?

The installer for older versions only installs webview2 if it isn't already present right?

```suggestion
WebView2 is supported on Windows 7 and newer and comes preinstalled on Windows 11. On versions older than Windows 11 the installer generated by Tauri takes care of installing WebView2 on the system if required.
```

File: docs/guides/webview-versions.md [hunk 6]
```diff
@@ -0,0 +1,190 @@
+# Webview Versions
+
+## WebView2 (Edge/Chromium)
```
Comment (lorenzolewis) [inline docs/guides/webview-versions.md]: Maybe have headings like this to make it easier to scan through? 

```suggestion
## Windows: WebView2
```

File: docs/guides/webview-versions.md [hunk 7]
```diff
@@ -0,0 +1,190 @@
...
+
+Tauri uses WebView2 which is based on Microsoft Edge and therefore Chromium. WebView2 can update itself, you are guaranteed a relatively recent chromium build on all Windows targets.
+
+WebView2 is supported on Windows 7 and newer and comes preinstalled on Windows 11. On versions older than Windows 11 the installer generated by Tauri takes care of ensuring WebView2 is installed on the system.
+
+## WebKit
```
Comment (lorenzolewis) [inline docs/guides/webview-versions.md]:
```suggestion
## macOS, iOS, & Linux: WebKit
```

File: docs/guides/architecture/webviews.md (hunk 2 above)
Comment (FabianLars) [reply docs/guides/architecture/webviews.md]: yep still relevant, gonna propose changes for a thing or 2 in a few.

File: docs/guides/webview-versions.md [hunk 8]
```diff
@@ -0,0 +1,190 @@
...
+
+Tauri uses WebKit on macOS (through [WKWebView](https://developer.apple.com/documentation/webkit/wkwebview?language=objc)) and on Linux (through webkit2gtk).
```
Comment (lorenzolewis) [inline docs/guides/webview-versions.md]:
```suggestion
Tauri uses WebKit on macOS (through [WKWebView](https://developer.apple.com/documentation/webkit/wkwebview?language=objc)) and on Linux (through `webkit2gtk`).
```

File: docs/guides/webview-versions.md [hunk 9]
```diff
@@ -0,0 +1,190 @@
...
+
+WebKit version numbers are made up of 5 segments and a numeric prefix indicating the OS WebKit is built for. The prefix is called the `SYSTEM_VERSION_PREFIX` and seems to be only present for macOS and iOS builds, not for Linux. Furthermore, if the last two segments are both `0` they can be omitted; So a version like `613.2.7.0.0` would be referred to as `613.2.7`.
+
+`$(SYSTEM_VERSION_PREFIX)$(MAJOR_VERSION).$(MINOR_VERSION).$(TINY_VERSION).$(MICRO_VERSION).$(NANO_VERSION)`
+
+### `SYSTEM_VERSION_PREFIX`
+
+Here is what the `SYSTEM_VERSION_PREFIX` values mean:
+
+| macOS version | SYSTEM_VERSION_PREFIX |
```
Comment (lorenzolewis) [inline docs/guides/webview-versions.md]: Generally I try to not put any heavily formatted things (like tables) in the disclosure areas because formatting and shading makes it a bit tougher to read. Maybe just don't put this in a disclosure tag and keep it fully inlined?

File: docs/guides/webview-versions.md [hunk 10]
```diff
@@ -0,0 +1,190 @@
...
+## WebKit
+
+Tauri uses WebKit on macOS (through [WKWebView](https://developer.apple.com/documentation/webkit/wkwebview?language=objc)) and on Linux (through webkit2gtk).
+
+Webkit version numbers are quite complicated, so below is some helpful information to understand them.
+
+<details>
+<summary>How to read WebKit version numbers</summary>
+
+WebKit version numbers are made up of 5 segments and a numeric prefix indicating the OS WebKit is built for. The prefix is called the `SYSTEM_VERSION_PREFIX` and seems to be only present for macOS and iOS builds, not for Linux. Furthermore, if the last two segments are both `0` they can be omitted; So a version like `613.2.7.0.0` would be referred to as `613.2.7`.
```
Comment (lorenzolewis) [inline docs/guides/webview-versions.md]:
```suggestion
WebKit version numbers are made up of 5 segments and a numeric prefix indicating which OS WebKit is built for. The numeric prefix is called the `SYSTEM_VERSION_PREFIX` and seems to be only present for macOS and iOS builds, not for Linux. Furthermore, if the last two segments are both `0` they can be omitted (so a version like `613.2.7.0.0` would be referred to as `613.2.7`).
```

File: docs/guides/webview-versions.md [hunk 11]
```diff
@@ -0,0 +1,190 @@
...
+The webkit version shipped with Safari 15.5 on macOS Monterey (12.x) has the following version number: `17613.2.7.1.8`.
+
+| part                    | version |
+| ----------------------- | ------- |
+| `SYSTEM_VERSION_PREFIX` | 17      |
+| `MAJOR_VERSION`         | 613     |
+| `MINOR_VERSION`         | 2       |
+| `TINY_VERSION`          | 7       |
+| `MICRO_VERSION`         | 1       |
+| `NANO_VERSION`          | 8       |
```
Comment (lorenzolewis) [inline docs/guides/webview-versions.md]: Format this as a bulleted list instead of a table may be a little easier to read (since there are so many tables on the page). I'd also move this directly up under line 20 where you're listing out the sections

File: docs/guides/webview-versions.md [hunk 12]
```diff
@@ -0,0 +1,190 @@
...
+| ------------- | --------------------- |
+| 10.11         | 11                    |
+| 10.12         | 12                    |
+| 10.13         | 13                    |
+| 10.14         | 14                    |
+| 10.15         | 15                    |
+| 11.0          | 16                    |
+| 12.0          | 17                    |
+| 13.0          | 18                    |
+| sdk=iphone\*  | 8                     |
```
Comment (lorenzolewis) [inline docs/guides/webview-versions.md]: Is the "*" for anything? I don't see something later in the page for it

File: docs/guides/webview-versions.md [hunk 13]
```diff
@@ -0,0 +1,190 @@
...
+| `SYSTEM_VERSION_PREFIX` | 17      |
+| `MAJOR_VERSION`         | 613     |
+| `MINOR_VERSION`         | 2       |
+| `TINY_VERSION`          | 7       |
+| `MICRO_VERSION`         | 1       |
+| `NANO_VERSION`          | 8       |
+
+</details>
+
+The tables below map OS versions to WebKit versions and the corresponding Safari equivalent, so you can use sites like [caniuse](https://caniuse.com) to figure out if a specific web platform feature is supported.
```
Comment (lorenzolewis) [inline docs/guides/webview-versions.md]:
```suggestion
The tables below map OS versions to WebKit versions and the corresponding Safari equivalent so that you can use sites like [caniuse](https://caniuse.com) to figure out if a specific web platform feature is supported.
```

File: docs/guides/webview-versions.md [hunk 14]
```diff
@@ -0,0 +1,190 @@
...
+
+### macOS
+
+On macOS Tauri uses the component that comes preinstalled with macOS since version 10.10 (Yosemite). It is considered a core component and is therefore updated with the regular OS updates. This means _unsuspported_ macOS versions also **do not** receive WebKit updates.
```
Comment (lorenzolewis) [inline docs/guides/webview-versions.md]:
```suggestion
On macOS Tauri uses the webview that comes preinstalled with macOS since version 10.10 (Yosemite). It is considered a core component and is therefore updated with the regular OS updates. This means _unsuspported_ macOS versions **do not** receive WebKit updates.
```

File: docs/guides/webview-versions.md [hunk 15]
```diff
@@ -0,0 +1,190 @@
...
+
+To find the Webkit version used by WKWebView you can use this command:
```
Comment (lorenzolewis) [inline docs/guides/webview-versions.md]:
```suggestion
To find the Webkit version used by `WKWebView` you can use this command in the terminal on macOS:
```

File: docs/guides/webview-versions.md (hunk 13 above)
Comment (lorenzolewis) [inline docs/guides/webview-versions.md]: Maybe move this just under the terminal command below? Between that and the table immediately after

File: docs/guides/webview-versions.md [hunk 16]
```diff
@@ -0,0 +1,190 @@
...
+
+```shell
+awk '/CFBundleVersion/{getline;gsub(/<[^>]*>/,"");print}' /System/Library/Frameworks/WebKit.framework/Resources/Info.plist
+```
+
+|             | macOS                            | WebKit           | Safari Equivalent | Notes                                                            |
```
Comment (lorenzolewis) [inline docs/guides/webview-versions.md]: Reverse the sort order of the table so that most recent is at the top. Can you "merge" cells vertically in a table so that all the rows in the first column are merged?

Comment (lorenzolewis) [inline docs/guides/webview-versions.md]: Use "OS Name | OS Version | ..." for the first two columns. Maybe move the "Safari Equivalent" column before webkit and rename it to "Safari Version" (also call it WebKit Version)?

File: docs/guides/webview-versions.md [hunk 17]
```diff
@@ -0,0 +1,190 @@
...
+| ----------- | -------------------------------- | ---------------- | ----------------- | ---------------------------------------------------------------- |
+| High Sierra | 10.13                            | [604.1.38.1.6]   | 11.0              |                                                                  |
+|             | 10.13.1                          | [604.3.5]        | 11.0.1            |                                                                  |
+|             | 10.13.2                          | [604.4.7.1.3]    | 11.0.2            | 27" iMacPro's received a special webkit version: [604.4.7.10.4]  |
```
Comment (lorenzolewis) [inline docs/guides/webview-versions.md]:
```suggestion
|             | 10.13.2                          | [604.4.7.1.3]    | 11.0.2            | 27" iMac Pro received a special WebKit version: [604.4.7.10.4]  |
```

File: docs/guides/webview-versions.md [hunk 18]
```diff
@@ -0,0 +1,190 @@
...
+|             | 10.13.2 Supplemental Update      | [604.4.7.1.6]    | 11.0.2            | 27" iMacPro's received a special webkit version: [604.4.7.10.6]  |
```
Comment (lorenzolewis) [inline docs/guides/webview-versions.md]: Grammatically these suggestions are weird to not have the plural and ownership, but that's Apple's guides 🤷🏽‍♂️

```suggestion
|             | 10.13.2 Supplemental Update      | [604.4.7.1.6]    | 11.0.2            | 27" iMac Pro received a special WebKit version: [604.4.7.10.6]  |
```

File: docs/guides/webview-versions.md [hunk 19]
```diff
@@ -0,0 +1,190 @@
...
+|             | 10.15.3                          | [608.5.11]       | 13.0.5            |                                                                  |
+|             | 10.15.4                          | [609.1.20.111.8] | 13.1              |                                                                  |
+|             | 10.15.5                          | [609.2.9.1.2]    | 13.1.1            |                                                                  |
+|             | 10.15.6                          | [609.3.5.1.3]    | 13.1.2            |                                                                  |
+|             | 10.15.7                          | [609.4.1]        | 13.1.3            |                                                                  |
+|             | 10.15.7 Security Update 2022-004 | [609.4.1.1.1]    |                   |                                                                  |
+| Big Sur     | 11.0                             | [610.2.11.1.3]   | 14.0.1            | Safari 14.0 was only ever available on iPhones                   |
```
Comment (lorenzolewis) [inline docs/guides/webview-versions.md]: Maybe be a bit more specific in the note and qualify it as "Safari 14.0.0"?

File: docs/guides/webview-versions.md [hunk 20]
```diff
@@ -0,0 +1,190 @@
...
+|             | 11.0.1                           | [610.2.11.51.8]  |                   |                                                                  |
+|             | 11.1                             | [610.3.7.1.9]    | 14.0.2            |                                                                  |
+|             | 11.2                             | [610.4.3.1.4]    | 14.0.3            |                                                                  |
+|             | 11.2.1                           |                  |                   |                                                                  |
+|             | 11.2.2                           |                  |                   |                                                                  |
+|             | 11.2.3                           | [610.4.3.1.7]    |                   |                                                                  |
+|             | 11.3                             | [611.1.21.161.3] | 14.1              | 24" M1 iMac's received a special webkit version: [611.1.21.1.12] |
```
Comment (lorenzolewis) [inline docs/guides/webview-versions.md]:
```suggestion
|             | 11.3                             | [611.1.21.161.3] | 14.1              | 24" M1 iMac received a special WebKit version: [611.1.21.1.12] |
```

File: docs/guides/webview-versions.md [hunk 21]
```diff
@@ -0,0 +1,190 @@
...
+[609.3.5.1.3]: https://github.com/WebKit/WebKit/blob/30fc8a44f087596c60e98adb434c0b98eccb61bb/Source/WebKit/Configurations/Version.xcconfig
+[609.4.1.1.1]: https://github.com/WebKit/WebKit/blob/8df64286794c38efa4697b7c24658cb85204a070/Source/WebKit/Configurations/Version.xcconfig
+[609.4.1]: https://github.com/WebKit/WebKit/blob/cb927e6151b5ef49c9ccfb13018f51471f8f1035/Source/WebKit/Configurations/Version.xcconfig
+[609.2.9.1.2]: https://github.com/WebKit/WebKit/blob/ca54d252f3416c3ec64f80a084cb5c4ff7ba24f1/Source/WebKit/Configurations/Version.xcconfig
+[609.1.20.111.8]: https://github.com/WebKit/WebKit/blob/5c90480a38a86464b6b421c2fd28c744b43a4faa/Source/WebKit/Configurations/Version.xcconfig
+[613.3.9.1.5]: https://github.com/WebKit/WebKit/blob/7f88b99524540e94abcdef4d45c1c0324d63fb56/Source/WebKit/Configurations/Version.xcconfig
+
+### Linux (WebKitGTK)
+
+The fractured nature of the Linux ecosystem means it is very hard to compile accurate information about WebKitGTK on the various distros. The table below is a very incomplete list of the most commonly used distributions and their webkit versions. You should always check your distros repositories for up-to-date information.
```
Comment (lorenzolewis) [inline docs/guides/webview-versions.md]:
```suggestion
The diverse nature of the Linux ecosystem means it is very hard to compile accurate information about WebKitGTK on the various distros. The table below is a very incomplete list of the most commonly used distributions and their WebKit versions. You should always check your distro's repositories for up-to-date information.
```

File: docs/guides/webview-versions.md [hunk 22]
```diff
@@ -0,0 +1,190 @@
...
+
+#### Note on the Ubuntu situation
```
Comment (lorenzolewis) [inline docs/guides/webview-versions.md]: Possibly also make this a :::note ::: thing instead of a separate subheading? 

```suggestion
#### Note On Ubuntu
```

File: docs/guides/webview-versions.md [hunk 23]
```diff
@@ -0,0 +1,190 @@
...
+
+On Ubuntu 18 through 20 it appears only `amd64` and `i386` machines received subsequent WebKit updates. All other supported arches remained on the initial releases' WebKit version. As you can see this results in really outdated Webkit versionson those machines. Since Ubuntu 22.04 however, it seems all arches receive WebKit updates.
+
+| Distro                                                         | webkitgtk | WebKit    | Safari Equivalent |
+| -------------------------------------------------------------- | --------- | --------- | ----------------- |
+| debian 9                                                       | 2.18      | [605.1.3] | 11.1              |
```
Comment (lorenzolewis) [inline docs/guides/webview-versions.md]:
```suggestion
| Debian 9                                                       | 2.18      | [605.1.3] | 11.1              |
```

File: docs/guides/webview-versions.md [hunk 24]
```diff
@@ -0,0 +1,190 @@
...
+[609.1.20.111.8]: https://github.com/WebKit/WebKit/blob/5c90480a38a86464b6b421c2fd28c744b43a4faa/Source/WebKit/Configurations/Version.xcconfig
+[613.3.9.1.5]: https://github.com/WebKit/WebKit/blob/7f88b99524540e94abcdef4d45c1c0324d63fb56/Source/WebKit/Configurations/Version.xcconfig
+
+### Linux (WebKitGTK)
+
+The fractured nature of the Linux ecosystem means it is very hard to compile accurate information about WebKitGTK on the various distros. The table below is a very incomplete list of the most commonly used distributions and their webkit versions. You should always check your distros repositories for up-to-date information.
+
+#### Note on the Ubuntu situation
+
+On Ubuntu 18 through 20 it appears only `amd64` and `i386` machines received subsequent WebKit updates. All other supported arches remained on the initial releases' WebKit version. As you can see this results in really outdated Webkit versionson those machines. Since Ubuntu 22.04 however, it seems all arches receive WebKit updates.
```
Comment (lorenzolewis) [inline docs/guides/webview-versions.md]:
```suggestion
On Ubuntu 18 through 20 it appears only `amd64` and `i386` machines received subsequent WebKit updates. All other supported arches remained on the initial releases' WebKit version. This results in outdated WebKit versions on those versions. Since Ubuntu 22.04 however, it seems all arches receive WebKit updates.
```

File: docs/guides/webview-versions.md (hunk 16 above)
Comment (lorenzolewis) [reply docs/guides/webview-versions.md]: Maybe also do something similar for "Safari Equivalent" where they would span multiple rows?

File: docs/guides/webview-versions.md (hunk 4 above)
Comment (FabianLars) [reply docs/guides/webview-versions.md]: > what powers

Nope. they are separate, though webview2 is basicallyyy a modified edge, or _used to be_. It's a little tricky to defined now that the can hardlink each other

> through OS updates

idk, that almost sounds to much like the monthly windows updates. webview2 updates separately from that, similar to how browser can update themselves (other than safari lol)

File: docs/guides/webview-versions.md (hunk 21 above)
Comment (FabianLars) [reply docs/guides/webview-versions.md]: diverse sounds too nice haha

Comment (lorenzolewis) [reply docs/guides/webview-versions.md]: Haha, you just want to add fuel to the fire! 🔥

Comment (FabianLars) [reply docs/guides/webview-versions.md]: I mean, i blamed the manjaro devs for our ssl cert issue today so yes i have a really special relationship with Linux lately :joy:

File: docs/guides/webview-versions.md [hunk 25]
```diff
@@ -0,0 +1,190 @@
...
+
+| Distro                                                         | webkitgtk | WebKit    | Safari Equivalent |
+| -------------------------------------------------------------- | --------- | --------- | ----------------- |
+| debian 9                                                       | 2.18      | [605.1.3] | 11.1              |
+| Ubuntu 18.04 (arm64, armhf, ppc64el, s390x)                    | 2.20      | [606.1.4] | 12.0              |
+| Ubuntu 20.04 (arm64, armhf, ppc64el, s390x)                    | 2.28      | [610.1.1] | 14.0              |
+| Ubuntu 18.04 (amd64, i386), centos 8 (non-stream)              | 2.32      | [612.1.6] | 15.0              |
+| debian 10                                                      | 2.34      | [613.1.1] | 15.4              |
+| Ubuntu 20.04 (amd64, i386), Ubuntu 22.04, debian 11, debian 12 | 2.36      | [614.1.6] | TP 140 (16.0)     |
```
Comment (FabianLars) [inline docs/guides/webview-versions.md]:
```suggestion
| Distro                                                         | webkitgtk | WebKit    | Safari Equivalent |
| -------------------------------------------------------------------------- | --------- | --------- | ------------- |
| Ubuntu 18.04                                                                                        | 2.20      | [606.1.4] | 12.0              |
| Debian 9 (with backport), Debian 10                                                      | 2.24      | [608.1.6] | 13.0              |
| Ubuntu 20.04                                                                                        | 2.28      | [610.1.1] | 14.0              |
| Debian 11, Ubuntu 18.04 (with update), centos 8 (non-stream)             | 2.32      | [612.1.6] | 15.0              |
| Debian 10 (with update)                                                                        | 2.34      | [613.1.1] | 15.4              |
| Debian 11 (with update), Ubuntu 20.04 (with update), Ubuntu 22.04    | 2.36      | [614.1.6] | TP 140 (16.0)     |
```

Debian 9 is a little special. webkitgtk version on release and in the normal update channel are too old, only with the _opt-in_ backports repo teh can get a version >=2.22 (which we set as minimum version in wry, tho later versions may still work idk).
p.s. debian 9 release = 2.16 - debain 9 update = 2.18

@JonasKruckenberg Do you still remember the file you mentioned where you got the webkit version from? Can't seem to find it but i need the webkit version for webkitgtk@2.24

**Review by FabianLars — CHANGES_REQUESTED** [review]

File: docs/guides/webview-versions.md (hunk 25 above)
Comment (JonasKruckenberg) [reply docs/guides/webview-versions.md]: webkitgtk@2.24 shipped webkit 608.1.6

Around mid-last year webkit started to switch to GitHub for version tracking, so for newer versions you have to check out the tags on GitHub. For older releases there is https://trac.webkit.org/browser specifically the releases "folder"

And then go to `Source/WebKit/Configurations/Version.xcconfig`

Comment (FabianLars) [reply docs/guides/webview-versions.md]: Thanks! ❤️ i updated the suggestion text.

File: docs/guides/webview-versions.md [hunk 26]
```diff
@@ -0,0 +1,186 @@
...
+| Debian 11, Ubuntu 18.04 (with update), centos 8 (non-stream)             | 2.32      | [612.1.6] | 15.0              |
+| Debian 10 (with update)                                                                        | 2.34      | [613.1.1] | 15.4              |
+| Debian 11 (with update), Ubuntu 20.04 (with update), Ubuntu 22.04    | 2.36      | [614.1.6] | TP 140 (16.0)     |
+
+[605.1.3]: https://trac.webkit.org/browser/webkit/releases/WebKitGTK/webkit-2.18/Source/WebKit/Configurations/Version.xcconfig
+[606.1.4]: https://trac.webkit.org/browser/webkit/releases/WebKitGTK/webkit-2.20/Source/WebKit/Configurations/Version.xcconfig
+[610.1.1]: https://trac.webkit.org/browser/webkit/releases/WebKitGTK/webkit-2.28/Source/WebKit/Configurations/Version.xcconfig
+[614.1.6]: https://trac.webkit.org/browser/webkit/releases/WebKitGTK/webkit-2.36/Source/WebKit/Configurations/Version.xcconfig
+[613.1.1]: https://trac.webkit.org/browser/webkit/releases/WebKitGTK/webkit-2.34/Source/WebKit/Configurations/Version.xcconfig
+[612.1.6]: https://trac.webkit.org/browser/webkit/releases/WebKitGTK/webkit-2.32/Source/WebKit/Configurations/Version.xcconfig
```
Comment (FabianLars) [inline docs/guides/webview-versions.md]: holy shit i messed up that formatting big time. github really needs monospace fonts in the suggestion dialog lol.
And i forgot the link, adding that now too:
```suggestion
| Distro                                                            | webkitgtk | WebKit    | Safari Equivalent |
| ----------------------------------------------------------------- | --------- | --------- | ----------------- |
| Ubuntu 18.04                                                      | 2.20      | [606.1.4] | 12.0              |
| Debian 9 (with backport), Debian 10                               | 2.24      | [608.1.6] | 13.0              |
| Ubuntu 20.04                                                      | 2.28      | [610.1.1] | 14.0              |
| Debian 11, Ubuntu 18.04 (with update), centos 8 (non-stream)      | 2.32      | [612.1.6] | 15.0              |
| Debian 10 (with update)                                           | 2.34      | [613.1.1] | 15.4              |
| Debian 11 (with update), Ubuntu 20.04 (with update), Ubuntu 22.04 | 2.36      | [614.1.6] | TP 140 (16.0)     |

[605.1.3]: https://trac.webkit.org/browser/webkit/releases/WebKitGTK/webkit-2.18/Source/WebKit/Configurations/Version.xcconfig
[606.1.4]: https://trac.webkit.org/browser/webkit/releases/WebKitGTK/webkit-2.20/Source/WebKit/Configurations/Version.xcconfig
[608.1.6]: https://trac.webkit.org/browser/webkit/releases/WebKitGTK/webkit-2.24/Source/WebKit/Configurations/Version.xcconfig
[610.1.1]: https://trac.webkit.org/browser/webkit/releases/WebKitGTK/webkit-2.28/Source/WebKit/Configurations/Version.xcconfig
[612.1.6]: https://trac.webkit.org/browser/webkit/releases/WebKitGTK/webkit-2.32/Source/WebKit/Configurations/Version.xcconfig
[613.1.1]: https://trac.webkit.org/browser/webkit/releases/WebKitGTK/webkit-2.34/Source/WebKit/Configurations/Version.xcconfig
[614.1.6]: https://trac.webkit.org/browser/webkit/releases/WebKitGTK/webkit-2.36/Source/WebKit/Configurations/Version.xcconfig
```

File: docs/guides/webview-versions.md (hunk 12 above)
Comment (JonasKruckenberg) [reply docs/guides/webview-versions.md]: That's taken straight from the WebKit Source 🤷‍♂️ I guess we can also remove that row entirely until we have proper iOS support

File: docs/guides/webview-versions.md (hunk 16 above)
Comment (JonasKruckenberg) [reply docs/guides/webview-versions.md]: I'm confused, what exactly do you mean?

**Comment by lorenzolewis** [comment]
@FabianLars and @JonasKruckenberg I moved this over to the References section and also did some formatting/rearranging if you all would like to take a look at it

**Review by FabianLars — APPROVED** [review]
